runtime/src/chpl-launcher-common.c

```diff
@@ -244,6 +244,28 @@ chpl_run_cmdstr(const char *commandStr, char *outbuf, int outbuflen) {
 }
 
 
+//
+// Find the named executable in the PATH, if it's there.
+//
+char *chpl_find_executable(const char *prog_name) {
+  const char *cmd_fmt = "/usr/bin/which --skip-alias --skip-functions %s";
```

Comment:
Why `/usr/ bin/which` instead of just `which`?

Reply:
In case the user has an alias or shell function with different behavior. I think we want to be using "classic which" rather than anything derivative.